### ISSUE TITLE
Has the mTCP development stopped?

### ISSUE THREAD

**Issue by sskras** [mtcp-stack/mtcp#337]
The last commit, https://github.com/mtcp-stack/mtcp/commit/0463aad5ecb6b5bca85903156ce1e314a58efc19 seems to be committed almost **5y ago**, on Nov 18, 2019.

Does that mean that project is abandoned or just postponed?

**Comment by megatog615** [comment]
No, I think it just doesn't have a lot going on here. Latest release on brutman.com is from January 10th of this year.

EDIT: I just realized this is not the same mTCP I was thinking of lol.

**Comment by sskras** [comment]
True, yours one targets DOS, and this one targets POSIX envs.

**Comment by gstrauss** [comment]
@ajamshed if further development is unlikely, please consider marking the repository as Archived.  Thank you.

https://docs.github.com/en/repositories/archiving-a-github-repository
https://docs.github.com/en/repositories/archiving-a-github-repository/archiving-repositories
